### ISSUE TITLE
Error when a player disconnects

### ISSUE THREAD

**Issue by FearMyShotz** [CodedRed-Spigot/PlayTimes#2]
```
[15:22:27 ERROR]: Could not pass event PlayerQuitEvent to PlaceholderAPI v2.10.9
java.lang.IllegalStateException: Cannot get plugin for class me.codedred.playtimes.PlayTimes from a static initializer
at org.bukkit.plugin.java.JavaPlugin.getPlugin(JavaPlugin.java:406) ~[patched_1.17.1.jar:git-Paper-370]
at me.codedred.playtimes.Expansions.getVersion(Expansions.java:73) ~[PlayTimes.jar:?]
at me.clip.placeholderapi.expansion.PlaceholderExpansion.hashCode(PlaceholderExpansion.java:226) ~[PlaceholderAPI-2.10.9.jar:?]
at com.google.common.collect.ImmutableSet$RegularSetBuilderImpl.add(ImmutableSet.java:742) ~[patched_1.17.1.jar:git-Paper-370]
at com.google.common.collect.ImmutableSet.construct(ImmutableSet.java:178) ~[patched_1.17.1.jar:git-Paper-370]
at com.google.common.collect.ImmutableSet.copyOf(ImmutableSet.java:212) ~[patched_1.17.1.jar:git-Paper-370]
at me.clip.placeholderapi.expansion.manager.LocalExpansionManager.getExpansions(LocalExpansionManager.java:115) ~[PlaceholderAPI-2.10.9.jar:?]
at me.clip.placeholderapi.expansion.manager.LocalExpansionManager.onQuit(LocalExpansionManager.java:352) ~[PlaceholderAPI-2.10.9.jar:?]
at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor65.execute(Unknown Source) ~[?:?]
at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.17.1.jar:git-Paper-370]
at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.17.1.jar:git-Paper-370]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.17.1.jar:git-Paper-370]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[patched_1.17.1.jar:git-Paper-370]
at net.minecraft.server.players.PlayerList.disconnect(PlayerList.java:607) ~[patched_1.17.1.jar:git-Paper-370]
at net.minecraft.server.players.PlayerList.disconnect(PlayerList.java:592) ~[patched_1.17.1.jar:git-Paper-370]
at net.minecraft.server.network.ServerGamePacketListenerImpl.onDisconnect(ServerGamePacketListenerImpl.java:1967) ~[app:?]
at net.minecraft.server.network.ServerGamePacketListenerImpl.onDisconnect(ServerGamePacketListenerImpl.java:1947) ~[app:?]
at net.minecraft.network.Connection.handleDisconnection(Connection.java:743) ~[app:?]
at net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:221) ~[app:?]
at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1653) ~[patched_1.17.1.jar:git-Paper-370]
at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:490) ~[patched_1.17.1.jar:git-Paper-370]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1480) ~[patched_1.17.1.jar:git-Paper-370]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1279) ~[patched_1.17.1.jar:git-Paper-370]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-370]
at java.lang.Thread.run(Thread.java:831) ~[?:?]
```

**Comment by CodedRedGIT** [comment]
Thanks for reporting, fixed in update 1.4.6. (Posting on Spigotmc.org soon)

**Comment by CodedRedGIT** [comment]
You can beta test the pre-release here: https://github.com/CodedRed-Spigot/PlayTimes/releases/tag/v1.4.6
